cogs/tickets.py

```diff
@@ -67,6 +67,9 @@ async def open_ticket(self, ctx, bot: discord.User):
             owner = ctx.guild.get_member(int(bot_db["owner"]["id"]))
             mods = ctx.guild.get_role(int(self.bot.settings["roles"]["mod"]))
             serverbots = ctx.guild.get_role(int(self.bot.settings['roles']['botpower']))
+            
+            if owner is None:
+                return await ctx.send("Bot owner is not in the server!")
```

Comment:
Please use our existing error format when sending errors, refer to the rest of the code.